docs/adr/0007-plugin-cloudformation-cli.md

```diff
@@ -0,0 +1,36 @@
+# Use AWS Command Line Interface for deployment
+
+For deploying a AWS CloudFormation template it has to be uploaded and stack has to be created on AWS.
+
+## Considered Alternatives
+
+* [AWS Command Line Interface](https://aws.amazon.com/de/cli/)
+* Manually with the AWS Management Console
+* Using API calls
+
+## Decision Outcome
+
+* Chosen Alternative: AWS Command Line Interface
+* Easiest and straight forward solution
+* AWS Command Line Interface has to be installed locally
+
+## Pros and Cons of the Alternatives <!-- optional -->
+
+### *AWS Command Line Interface*
+
+`+` Easy to use
+`+` Easy to install
```

Comment:
Wouldn't consider this a Pro because you list the point that the CLI has to be installed (and probably also be configured) under Cons. Thats kind of contradictory.